cpu_map_keyme2560.h

```diff
@@ -61,7 +61,8 @@
 #define C_DISABLE_BIT 5  // J5 /Atmega Pin 68
 #define STEPPERS_DISABLE_MASK ((1<<X_DISABLE_BIT)|(1<<Y_DISABLE_BIT)|(1<<Z_DISABLE_BIT)|(1<<C_DISABLE_BIT))   // All disable bits
 
-#define STEPPERS_LONG_LOCK_MASK  ((1<<Y_DISABLE_BIT)|(1<<Z_DISABLE_BIT))  //Keep gripper and Y engaged longer
+//Keep gripper, carousel, and Y engaged longer
+#define STEPPERS_LONG_LOCK_MASK  ((1<<Y_DISABLE_BIT)|(1<<Z_DISABLE_BIT)|(1 << C_DISABLE_BIT))
```

Comment:
I intentionally try not to mix stylistic changes with functional changes within the same statement.